lib/tests/auth/cognito.test.ts

```diff
@@ -120,7 +120,7 @@ describe("Test Cognito library", () => {
         id: mockedId,
       });
 
-      mockGenerateVerificationCode.mockReturnValueOnce("a1é3_8");
```

Comment:
I know the value is purely contained within the test but I wonder if we should only use the characters available for an actual 2FA code?

Reply:
Sure I can change it :)